--- xf86-video-trident-1.3.6/src/trident_driver.c.0000~	2012-07-16 04:16:00.000000000 +0200
+++ xf86-video-trident-1.3.6/src/trident_driver.c	2013-04-06 02:36:30.068954305 +0200
@@ -56,7 +56,6 @@
 
 #include "mipointer.h"
 
-#include "mibstore.h"
 #include "shadow.h"
 #include "trident.h"
 #include "trident_regs.h"
@@ -3037,7 +3036,6 @@ TRIDENTScreenInit(SCREEN_INIT_ARGS_DECL)
     	TridentAccelInit(pScreen);
     }
 
-    miInitializeBackingStore(pScreen);
     xf86SetBackingStore(pScreen);
 
     /* Initialise cursor functions */
